### ISSUE TITLE
Unmatched terms between `bioentry` and `bioentry.accession`?

### ISSUE THREAD

**Issue by Asutu** [biosql/biosql#25]
Hi,

I'm having a bit of a rough time trying to understand the reason for the following behaviour:

```
$ sqlite3 nt.db "SELECT * FROM bioentry LIMIT 1;"
1|1|7845|XM_019431108|XM_019431108|XM_019431108.1|MAM|PREDICTED: Panthera pardus proline rich 11 (PRR11), transcript variant X5, mRNA|1
```

```
$ sqlite3 nt.db 'SELECT bioentry.accession from bioentry LIMIT 10'
AB091111
```

In the second query, I'd expect to see "XM_019431108", the accession for [PREDICTED: Panthera pardus proline rich 11 (PRR11), transcript variant X5, mRNA](https://www.ncbi.nlm.nih.gov/nuccore/XM_019431108.1/), but instead I'm seeing "AB091111", which is the accession for [Cyprinus carpio IgL2 mRNA for Immunoglobulin light chain, partial cds, clone:Cph2](https://www.ncbi.nlm.nih.gov/nuccore/AB091111.1/).

I'm not understanding why the two queries give distinct results and am worried that something is not right in my database.

Some background:
- I'm using `sqlite3`, so for some of the tasks below I had to adapt some commands to this implementation (for example, in contrast with mysql or postgres, for which the examples were usually made)
- loaded the taxonomy as per BioSQL instructions using the `load_ncbi_taxonomy.pl` script
- I'm just using a subset of the NCBI `nt` database, currently with a random set of 1000 identifiers
- the BioSQL database is filled with these identifiers using BioPython as per the instructions detailed [here](https://biopython.org/wiki/BioSQL) (basically with `Entrez.efetch` to fetch genbank records for the identifiers and then loading these into the database)
- my random subset of the NCBI `nt` database, includes both the identifiers "XM_019431108" and "AB091111"

Any insights that could shed some light on the behaviour reported?

Many thanks,
Pedro

**Comment by hlapp** [comment]
@Asutu it looks like you're relying on the assumption that a query without any WHERE clause will consistently return the same record as the first row in the result. This is normally not true. Also, your first query only seems to contain one field (despite a `SELECT *`) and your second query only one result row (despite a `LIMIT 10`). Neither makes any sense to me if you have more than record in the bioentry table.

So it looks like you're not pasting the full output?

Regardless however, this doesn't seem to be an issue with the database model itself but more with how the software you are using (biopython) populates it? This would then be a question to ask under that software repo's (i.e., the Biopython) tracker.

**Comment by Asutu** [comment]
Yes, in the second case the query shows the expected 10 results but I mistakenly only copied the first line for comparison with the first query. Limiting the second query to `1` returns the same accession  as in question (AB091111).

Thanks for the clarification. I think I'm still a bit confused with this, although when I select for either of the accessions, the results are correct which suggests that the database is set properly.
